src/libos/src/config.rs

```diff
@@ -200,6 +203,20 @@ impl Config {
         };
         let feature = ConfigFeature::from_input(&input.feature)?;
 
+        if input.disable_log {
+            log::set_max_level(LevelFilter::Off);
+        } else if !sgx_allow_debug() {
+            if log::max_level() == LevelFilter::Trace {
```

Comment:
According to the previous discussion, only error logs can be output in release mode.
`log::set_max_level(LevelFilter::Error);`